lua/autorun/client/ttt_bind_autorun.lua

```diff
@@ -1,3 +1,4 @@
+if engine.ActiveGamemode() ~= "terrortown" then return end--block if TTT is not running
 ------------
```

Comment:
This file should not be moved and the `ttt_bind_autorun.lua` file should still be kept, as it will add the lua file to the files a client has to download.

Reply:
Why? I don‘t see any reason, autorun files are downloaded automatically afaik

Reply:
Yes, but we do not have "logic-heavy" files in the autorun folder as of now, and i think that is much cleaner as it is currently, with the bind code in its own folder.